### ISSUE TITLE
Suggestions for slide deck template

### ISSUE THREAD

**Issue by terichadbourne** [JiggyPete/global-diversity-cfp-day-site#45]
Having a slide deck template to start from was wonderful, and I loved the little links to go out to the right videos on YouTube!

Dropping notes here on a few things we edited for our Boston slides that could be considered for edits  to the global template. Clearly every team will need to customize regardless, but if there are any of these that you'd like to treat as best practices, adding to the deck will encourage local organizers to think about them. 

- Add a slide thanking sponsors.
- Add a slide about the CoC early in the deck.
- Add a slide about helping each other by adding pronoun stickers and and anti-photography stickers to badges, assuming positive intent and helping each other learn.
- Before the CFP Advice video/preso, add a slide on what "CFP" means. We called up various interpretations used in different industries (papers, posters, participation) and landed on Call for Presentations to define what we'd be talking about in this workshop.
- Add event hashtags / handles in a header or footer applied to all slides.
- Add a slide near the end of deck to put up while collecting survey feedback with a "How did we do?" header. (We used paper feedback forms for the highest response rate and put explicit time in the schedule for this activity, which worked well. We took the family photo before doing surveys so that everyone could take their time and leave whenever they finished.) 

Feel free to ignore any of these suggestions if they feel too one-off to you.

**Comment by JiggyPete** [comment]
So much goodness here @terichadbourne, thank you!

**Comment by terichadbourne** [comment]
Of course! Happy to email you our version of the deck if it would be helpful @JiggyPete. Just let me know.

**Comment by JiggyPete** [comment]
@terichadbourne starting to ramp back up and open our Call for Organisers / Workshop Teams over the next few months. If you could send over the template (if you still have it) that'd be much appreciated

**Comment by terichadbourne** [comment]
@JiggyPete Just found our keynote deck and emailed it to you. Hope it's helpful!

**Comment by JiggyPete** [comment]
Thanks Teri, that's great! 🙏🏼

**Comment by JiggyPete** [comment]
Last thing to do is integrate Teri's deck and suggestions with the one that is downloadable from the CFP Day site
